### ISSUE TITLE
Generation of ...ImplBase class can produce invalid kotlin with certain comment patterns

### ISSUE THREAD

**Issue by wfhartford** [rouzwawi/grpc-kotlin#35]
So I've been trying to upgrade to the latest version of grpc and ran into an amusing issue with an imported proto file. Put in the simplest case, the following service definition will result in an `ImplBase` class which does not compile:

```
service Blah {
  // /*
  rpc Yabadabadoo(Request) returns (Response);
}
```

The issue in the comment where the resulting code will look something like this:

```kotlin
abstract class BlahImplBase(
    coroutineContext: CoroutineContext = Dispatchers.Default
) : BindableService, CoroutineScope {

  /**
      /*
  */
  open suspend fun yabadabadoo(...
}
```

An interesting fact that I just learned is that, unlike Java, Kotlin supports nested block comments, which means that the comment on the `yabadabadoo` method above puts the entire rest of the class inside the documentation comment.

While this is a contrived example, and a reasonable response would be _well then don't do that_, this pattern appears in some commonly imported protobuf files provided by google, such as the following excerpt from the `google/longrunning/operations.proto` in `com.google.api.grpc:proto-google-common-protos:1.12.0`:

```
  // Lists operations that match the specified filter in the request. If the
  // server doesn't support this method, it returns `UNIMPLEMENTED`.
  //
  // NOTE: the `name` binding below allows API services to override the binding
  // to use different resource name schemes, such as `users/*/operations`.
  rpc ListOperations(ListOperationsRequest) returns (ListOperationsResponse) {
    option (google.api.http) = { get: "/v1/{name=operations}" };
  }
```
The string `user/*/operations` towards the end starts a nested block comment.

**Comment by wfhartford** [comment]
Unless any high priorities come up, I will work on a PR for this tomorrow. The solution that comes to mind is to preprocess any comments inserted into the code and replace instances of `/*` with `/&#42;`.

**Comment by rouzwawi** [comment]
Resolved by #37
